src/Components/DraftFeed/Components/DraftablePlayers/DraftablePlayers.js

```diff
@@ -0,0 +1,31 @@
+import './DraftablePlayers.css';
+import React, { useState } from 'react';
+import { NavLink } from 'react-router-dom';
+export var players = [];
+
+export function DraftablePlayers({ name, position, FFP }) {
```

Comment:
This name should probably be `DraftablePlayer` since it represents a single player

Reply:
Resolved